test/scaling/scaling.jl

```diff
@@ -11,7 +11,7 @@ using Test, LinearAlgebra, StaticArrays
     @test typeof(sitp) <: Interpolations.ScaledInterpolation
     @test parent(sitp) === itp
 
-    for (x,y) in zip(-3:.05:1.5, 1:.1:10)
+    for (x,y) in zip(-3:.05:1.5, 1:.1:10, LinRange(-3,1.5,91))
```

Comment:
I should use the code review functionality. 😃 This should be
```
for (x,y) in zip(LinRange(-3,1.5,91), 1:.1:10)
```

Reply:
Ah. Yes, Clearly I did not read that. But we are still not there, right, because we don't want to lose the original test that works with StepRangeLen. I will need to add another one.